### DISCUSSION TITLE
wiki behind reverse proxy

### DISCUSSION THREAD

**Discussion by frefal** [requarks/wiki#5012] [Help / Questions] [answered]
I have the wiki setup on a machine behind a reverse proxy. The reverse proxy is sending the incoming requests from https://domain/wiki to http://192.168.1.10:3000

Within the wiki page, under administration I've setup the site URL as https://domain/wiki

Now when you try to open up https://domain/wiki it fails, as the wiki is telling the client to fetch content under https://domain/ and not https://domain/wiki/

Version is 2.5.274

Any suggestions would be greatly appreciated!

**Comment by justmvv** [comment, marked as answer]
Same trouble. I see there is WIKI.ROOTPATH variable in code. If I define WIKI.ROOTPATH='/wiki' does it change root from '/' to '/wiki'? 

P.S. The question was answered here https://github.com/Requarks/wiki/discussions/4656

**Reply by frefal** [reply]
Thanks for the info there. Too bad it won't work using a subpath. using a subdomain is not a solution for me.

**Reply by mikhail-khoroshev** [reply]
Look at this thread https://github.com/Requarks/wiki/discussions/3501.
